test/test.js

```diff
@@ -1765,3 +1765,30 @@ test( "Validation triggered on radio and checkbox via click", function() {
 	// test if there is no error anymore
 	ok( form.find( "input.error" ).length === 0, "Form valid" );
 });
+
+test( "destruction of Validator", function() {
+	expect( 2 );
+
+	var form = $( "#validatorDestroy" ),
+		validate = form.validate();
+
+	form.on( "submit.validate", function( event )
+	{
+		event.preventDefault();
+
+		ok( !!$( this ).data( "validator" ), "Validator has events and object" );
+	});
+
+	form.submit();
+
+	validate.destroy();
+
+	form.on( "submit.validate", function( event )
```

Comment:
That is indeed a good point, however, the reason I had chosen to do it this way is that the only times I had encountered errors was when I was submitting the form just after I had reset it - and some events were still active. I can change it to the way you suggested it to be if you'd like?